saas/doc/changelog.rst

```diff
@@ -1,3 +1,8 @@
+`1.0.2`
+-------
+
+- **Fix:** fix db name vulnerability
```

Comment:
What is that?

Reply:
Changelog message?

Reply:
I mean I don't understand what does it fix

Reply:
after the user submits the name for the database, it will be processed so that it can be used in the link

Reply:
ok, I got the idea, but it doesn't come from your description. Please clarify changelog (e.g. "make deafult db name compatible with new slugify function")